includes/admin.form.inc

```diff
@@ -75,3 +90,23 @@ function islandora_audio_admin(array $form, array &$form_state) {
 function islandora_audio_admin_form_lame_ajax_callback(array $form, array $form_state) {
   return $form['islandora_lame_url'];
 }
+
+/**
+ * Validation callback for the VBR Quality field.
+ *
+ * @param array $element
+ *   The Drupal element definition.
+ * @param array $form_state
+ *   The Drupal form state.
+ */
+function islandora_audio_vbr_quality_validate(array $element, array &$form_state) {
+  if (!empty($element['#value'])) {
+    if ($element['#value'] < 0 || $element['#value'] > 9.99) {
```

Comment:
Should this be `> 9.999`? Not that anyone should ever be setting the quality that low (bleh) ... just would be funny if someone submitted the form and was all "but 9.994 _is_ lower than 9.999!"

That scenario started sounding more and more unlikely as I typed it out